v3/spring-cloud-aws-v3-autoconfigure/src/main/java/io/awspring/cloud/v3/autoconfigure/ses/properties/AwsSesProperties.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2013-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.awspring.cloud.v3.autoconfigure.ses.properties;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Properties related to AWS Simple Email Service.
+ *
+ * @author Eddú Meléndez
+ * @author Arun Patra
+ */
+@ConfigurationProperties(prefix = AwsSesProperties.PREFIX)
+public class AwsSesProperties {
```

Comment:
we don't use `properties` package. this class should be at the same level as `SesAutoConfiguration`, package `io.awspring.cloud.v3.autoconfigure.ses.`. Also rename it to `SesProperties`